### PR TITLE
Create PRS13233DerarWaldaQirqos.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
scribe MY-023

### CODE REVIEW

File: new/PRS13233DerarWaldaQirqos.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <change who="DN" when="2020-11-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ድራር፡ ወልደ፡ ቂርቆስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dǝrār Walda Qirqos</persName>
+                    <persName type="alt"><roleName type="title">ʾalaqā</roleName>Dǝrār Walda Qirqos</persName>
```
Comment (eu-genia) [inline new/PRS13233DerarWaldaQirqos.xml:50]: you do not need to repeat the name here - or if you do insert a blank space before

File: new/PRS13233DerarWaldaQirqos.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
```
Comment (eu-genia) [inline new/PRS13233DerarWaldaQirqos.xml:55]: empty tags

File: new/PRS13233DerarWaldaQirqos.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    <floruit notBefore="1965" notAfter="2020"></floruit>
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS13233DerarWaldaQirqos" passive="ESmy023"/>
+                    <relation name="saws:isRelatedTo" active="PRS13233DerarWaldaQirqos" passive="LOC7348Legat"/>
```
Comment (eu-genia) [inline new/PRS13233DerarWaldaQirqos.xml:60]: what is the relationship to Legat? can you add the info as text to the floruit line?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13233DerarWaldaQirqos.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13233DerarWaldaQirqos.xml:50]: ok

File: new/PRS13233DerarWaldaQirqos.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS13233DerarWaldaQirqos.xml:55]: ok

File: new/PRS13233DerarWaldaQirqos.xml [hunk 4]
```diff
@@ -47,13 +47,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                    <persName type="alt"><roleName type="title">ʾalaqā</roleName>Dǝrār Walda Qirqos</persName>
+                    <persName type="alt"><roleName type="title">ʾalaqā</roleName> Dǝrār Walda Qirqos</persName>
                     <faith type="EOTC"/>
                     <occupation type="ecclesiastic"/>
                     <nationality type="Ethiopia"/>
-                    <birth/>
-                    <death/>
-                    <floruit notBefore="1965" notAfter="2020"></floruit>
+                    <floruit notBefore="1965" notAfter="2020">A scribe who resided and worked at the 
+                    village of Lǝgat.</floruit>
```
Comment (eu-genia) [inline new/PRS13233DerarWaldaQirqos.xml:55]: `village of <placeName ref="LOC7348Legat">Lǝgat</placeName>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13233DerarWaldaQirqos.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS13233DerarWaldaQirqos.xml:59]: Done, like this?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**